letsencrypt/client/ddns/ddns.py

```diff
@@ -0,0 +1,86 @@
+"""generic DDNS authenticator (RFC 2136)"""
+import logging
+import subprocess
+
+import zope.interface
+
+from letsencrypt.client import challenge_util
+from letsencrypt.client import interfaces
+
+
+# TODO: make practical use of this, nothing uses DDNS yet
+class DDNS(object):
+    """DDNS authenticator.
+
+    """
```

Comment:
Still extra whitespace on 14-15.  I can take care of this on the eventual merge if you don't end up taking another pass.